index.js

```diff
@@ -0,0 +1,57 @@
+var express = require('express');
+var fs = require('fs');
+var bodyParser = require('body-parser');
+var app = express();
+
+app.set('view engine', 'ejs');
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(express.static('static_assets'));
+
+app.get('/', function(req, res) {
+  res.render('index');
+});
+
+app.get('/site/about', function(req, res){
```

Comment:
typically you would just use /about or /contact for a site related page but you kept the root context appropriate which is the important thing.